### PR TITLE
Bug 1882694: Pin OVS to latest 4.4.22 RPM version

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
As done in https://github.com/openshift/ovn-kubernetes/pull/284 but for OVS

/assign @dcbw @trozet 

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by alexanderConstantinescu** [comment]
/retitle Bug 1882694: [release-4.5] Pin OVS to latest 4.4.22 RPM version

File: Dockerfile [hunk 1]
```diff
@@ -39,11 +39,10 @@ RUN INSTALL_PKGS=" \
 	tcpdump \
 	" && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && \
-	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "openvswitch2.13 >= 2.13.0-29.el7fdp" openvswitch2.13-devel && \
-	# OVN-Kubernetes 4.4.24 comes shipped with these OVN versions.
 	#======== 4.4.24 version ========
 	# ovn2.13-20.06.2-4.el7fdp.x86_64
-	# We need to pin to the 4.4.24 version
+	# openvswitch2.13-2.13.0-46.el7fdp.x86_64
+	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "openvswitch2.13 >= 2.13.0-46.el7fdp" "openvswitch2.13-devel == 2.13.0-46.el7fdp" && \
```
Comment (trozet) [inline Dockerfile:45]: you have  >= here, shoudln't that be == ?

**Review by trozet — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (alexanderConstantinescu) [reply Dockerfile]: It should. Thanks for spotting that!

**Comment by alexanderConstantinescu** [comment]
> /retest

How about an `/lgtm` and `/approve` and we let the bot do that for us? :smiling_imp:

**Comment by alexanderConstantinescu** [comment]
/hold

this PR has been on the back-burner for too long. OVS on 4.4.z and 4.5.z have been bumped to 2.13.0-50.el7fdp. And moreover, I this PR needs to bump OVS to 2.13.0-63.el7fdp as to get the CVE fix in directly. But first we will need https://github.com/openshift/ovn-kubernetes/pull/358 to get in and have master and 4.6 on that version

**Comment by crawford** [comment]
@alexanderConstantinescu any update?

**Pull request merged**